dateutil/parser.py

```diff
@@ -1328,9 +1327,6 @@ def _parse_hms(i, l, info, res):
     return i
 
 
-# TODO: require len(token) >= 3 like we do for the between-parens version?
```

Comment:
Note: I removed this because I don't think we can require `len(token) >= 3` here, since `Z` is most definitely a valid potential tzname.